src/backend/direct.rs

```diff
@@ -697,6 +701,8 @@ impl crate::Context for Context {
         mode: MapMode,
         range: Range<wgt::BufferAddress>,
     ) -> Self::MapAsyncFuture {
+        wgc::span!(_guard, TRACE, "Buffer::buffer_map_async wrapper");
```

Comment:
dummy question: why are we even tracing wgpu-rs? Most methods here are just pass-through to wgpu-core, which is already traced

Reply:
These exist so that non trivial wrappers show up in the chrome tracing trace. This information isn't very useful for logging, so I reduced it to the maximum verbosity.